### ISSUE TITLE
"Failed to get game data from the server."

### ISSUE THREAD

**Issue by intelxp** [finndev/PokeBuddy#1006]
I have try to login on iPhone apps, but show this "Our servers are humbled by your incredible response. We are working to resolve this issue. Please try again soon!" , okay, I try again, but this times is show "Failed to get game data from the server." , I have retry more but not work, maybe my account data have been deleted from server. :(

**Comment by AlexCtasy** [comment]
@intelxp Yes dude, considere your self perma banned.

**Comment by futi666** [comment]
same here with my lvl 34 acc... perma ban :/

**Comment by iPetrov2** [comment]
@intelxp i havent had that problem yet. ill wait for about a day to see if i got perma banned XD. the only problem i have is i cant catch pokemon which im pretty sure is just a soft ban

**Comment by NegroSorete** [comment]
you got perma ban

**Comment by iPetrov2** [comment]
@NegroSorete me?

**Comment by AlexCtasy** [comment]
@iPetrov2 YEs petro, u got perma also, dont waste your time. Create another account and go bot with necrobot xoaoxaoxa, that softban u have, days ago u fix u just spin pokestops for 40 times.. But right now forget about it.
